tasks/move-granules/index.js

```diff
@@ -7,12 +7,7 @@ const flatten = require('lodash/flatten');
 const keyBy = require('lodash/keyBy');
 const path = require('path');
 
-const {
-  moveObject,
-  s3Join,
-  s3ObjectExists,
-  waitForObjectToExist,
-} = require('@cumulus/aws-client/S3');
+const S3 = require('@cumulus/aws-client/S3');
```

Comment:
Not that this is a merge conflict, but this doesn't really seem like a great change.